src/vm/threads.h

```diff
@@ -3721,7 +3721,7 @@ class Thread: public IUnknown
     ARG_SLOT CallPropertyGet(BinderMethodID id, OBJECTREF pObject);
     ARG_SLOT CallPropertySet(BinderMethodID id, OBJECTREF pObject, OBJECTREF pValue);
 
-#if defined(FEATURE_HIJACK) && !defined(PLATFORM_UNIX)
+#if defined(FEATURE_HIJACK) && !defined(FEATURE_PAL)
```

Comment:
@janvorli Oh, I'll revert just as before if there is a plan to revise this macro. Thanks you for comment!